refund_test.go

```diff
@@ -3,69 +3,27 @@ package stripe
 import (
 	"encoding/json"
 	"testing"
-)
-
-func TestRefundUnmarshal(t *testing.T) {
-	refundData := map[string]interface{}{
-		"id":     String("re_1234"),
-		"object": String("refund"),
-		"charge": String("ch_1234"),
-	}
-
-	bytes, err := json.Marshal(&refundData)
-	if err != nil {
-		t.Error(err)
-	}
-
-	var refund Refund
-	err = json.Unmarshal(bytes, &refund)
-	if err != nil {
-		t.Error(err)
-	}
-
-	if refund.ID != "re_1234" {
-		t.Errorf("Problem deserializing refund, got ID %v", refund.ID)
-	}
-
-	if refund.Charge == nil {
-		t.Errorf("Problem deserializing refund, didn't get a Charge")
-	}
 
-	if refund.Charge.ID != "ch_1234" {
-		t.Errorf("Problem deserializing refund.charge, wrong value for ID")
-	}
-}
-
-func TestRefundUnmarshalExpandedCharge(t *testing.T) {
```

Comment:
I'm guessing you removed those on purpose because we assume that this is tested by the charge's `UnmarshalJSON`?

Reply:
Yeah, I think what's in the `UnmarshalJSON` is "good enough". It tests the important parts of unmarshaling to an object, but more succinctly.